citations.bib

```diff
@@ -164,4 +164,34 @@ @TECHREPORT{Hassett:2016
   type = {Congressional Testimony},
   month = {February},
   url = {https://www.aei.org/wp-content/uploads/2016/02/KHtestimony.pdf}
+}
+
+@TECHREPORT{DEP:2021a,
+  AUTHOR = {Jason DeBacker and Richard W. Evans and Benjamin R. Page},
+  TITLE = {A Detailed Macroeconomic Analysis of President Biden's 2020 Campaign Tax Proposals},
+  INSTITUTION = {Tax Policy Center},
+  YEAR = {2021},
+  type = {Working Paper},
+  month = {July},
+  url = {https://www.taxpolicycenter.org/publications/detailed-macroeconomic-analysis-president-bidens-2020-campaign-tax-proposals}
+}
+
+@TECHREPORT{DEP:2021b,
+  AUTHOR = {Jason DeBacker and Richard W. Evans and Benjamin R. Page},
+  TITLE = {A Sensitivity Analysis of a Detailed Macroeconomic Analysis of President Biden's 2020 Campaign Tax Proposals},
+  INSTITUTION = {Tax Policy Center},
+  YEAR = {2021},
+  type = {Working Paper},
+  month = {July},
+  url = {https://www.taxpolicycenter.org/publications/sensitivity-analysis-detailed-macroeconomic-analysis-president-bidens-2020-campaign-tax}
+}
+
+@TECHREPORT{DEP:2021b,
```

Comment:
@jdebacker, I think the error in the build CI test will be resolved if you change this reference to `DEP:2021c`.